Mobl_ios_Callisto-master/MealTrackingPilot/ChatBot/ChatViewController.swift

```diff
@@ -103,6 +103,8 @@ class ChatViewController: UIViewController, UITableViewDataSource, UITableViewDe
         tableView.delegate = self
 
         tableView.tableFooterView = UIView(frame: .zero)
+        
+        tableView.transform = CGAffineTransform(scaleX: 1, y: -1)
```

Comment:
Please use below for transform,
tableView.transform = CGAffineTransform(rotationAngle: (-.pi))